WordPress/src/main/java/org/wordpress/android/ui/posts/ViewLayoutType.kt

```diff
@@ -0,0 +1,16 @@
+package org.wordpress.android.ui.posts
+
+enum class ViewLayoutType(val id: Long) {
```

Comment:
Sure, OK. I don't mind general too much in general (no pun intended :D) for Android because it's namespaced anyway as opposed to iOS where the names end up really long, but if you'd prefer it given a name to fit with its namespace that's all good. I don't have a strong preference really.

Reply:
>I don't mind general too much in general (no pun intended :D)
:D

> if you'd prefer it given a name to fit with its namespace that's all good.

This might be my personal preference, rather than a general practice in our code base. However, I feel the autocompletion can become a bit useless as there are more classes with general names - it's also pretty easy to unintentionally import a wrong class.

I'll leave the final decision to you. I don't feel strongly about it.